.neutrinorc.js

```diff
@@ -96,6 +189,8 @@ module.exports = {
           .test(/JSONStream/)
           .use('shebang')
             .loader('shebang-loader');
+
+      console.log(JSON.stringify(neutrino.config.toConfig(), null, 2));
```

Comment:
Manually stringifying the config is not neccessary/gives worse output. Try:
````
neutrino --inspect --mode {development,production}
# or
NODE_ENV={development,production,test} neutrino --inspect
```

(https://master.neutrinojs.org/usage/#inspecting-the-generated-webpack-config)

Reply:
It looks like the globally `yarn link`ed version of Neutrino is using an older version of webpack-chain. I would either:
1. run `yarn neutrino ...` (in the hope that it favours the copy of Neutrino in the local `node_modules`)
2. remove the yarn link for neutrino
3. update the `neutrino-dev` source directory to `master` and run `yarn` to install the latest version of deps

(I would suggest 1/2, otherwise we'll be debugging Neutrino `master` rather than the known release version, which is not what `yarn start` will be using)

Reply:
Created https://github.com/neutrinojs/neutrino/issues/1167.

Reply:
I'm not sure I am running a linked version of Neutrino. Running `grep "webpack-chain" node_modules/neutrino/package.json` shows `"webpack-chain": "^4.11.0",`. The update to webpack-chain was merged [two days ago](https://github.com/neutrinojs/neutrino/commit/84bdf6a08383bfd69d7f3c93f0e48117893ce84e). Is it possible that it hasn't been published yet?

Reply:
If `neutrino` is on `PATH` (ie `neutrino` works and not just `yarn neutrino`), it's a linked copy, or else `PATH` has been manually adjusted.

That grep command is a red herring, since that will be grepping the local node_modules copy, not the copy that's being used from the linked neutrino.

Try running this:
`ls -al $(yarn global bin)/neutrino`

Reply:
Removing the linked version of Neutrino works! Many thanks!

Reply:
Awesome :-)